scripts/cli/README.md

```diff
@@ -41,149 +44,20 @@ The test scripts include those named below, and most make up `npm test`:
-	Block Number: 6		Processed: 2	Time (ms): 6031
-	Block Number: 7		Processed: 2	Time (ms): 5966
-	Block Number: 8		Processed: 32	Time (ms): 6004
-	Block Number: 9		Processed: 32	Time (ms): 5998
-DONE
-Claims Batch Test
-Claims length: 10
-Claim Batch Test Completed
-```
-
-### Fast Run
-
-NB - note failures in DID creation and token issuance as the "demo" namespace was reused
-
-```
-$ node ./index.js -n 30 -t 5 -c 10 -p demo -d /tmp/pmesh-primary-node -f
-Multiple versions of @polkadot/keyring detected, ensure that there is only one version in your dependency tree
-Welcome to Polymesh Stats Collector. Creating 30 accounts and DIDs, with 10 claims per DID.
-Unknown types found, no types for MaybeVrf
-Initial storage size (/tmp/pmesh-primary-node): 21.9453125MB
-Generating Primary Keys
-Generating Secondary Keys
-Generating Claim Keys
-=== Processing Transactions ===
-████████████████████████████████████████ | Submit  : TPS                             | 30/30
-░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | Complete: TPS                             | 0/30
-████████████████████████████████████████ | Submit  : DISTRIBUTE POLY                 | 65/65
-░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | Complete: DISTRIBUTE POLY                 | 0/65
-████████████████████████████████████████ | Submit  : CREATE ISSUER IDENTITIES        | 30/30
-░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | Complete: CREATE ISSUER IDENTITIES        | 0/30
-████████████████████████████████████████ | Submit  : ADD SECONDARY KEYS                | 30/30
-░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | Complete: ADD SECONDARY KEYS                | 0/30
-████████████████████████████████████████ | Submit  : SET SECONDARY KEY ROLES           | 30/30
-░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | Complete: SET SECONDARY KEY ROLES           | 0/30
-████████████████████████████████████████ | Submit  : ISSUE SECURITY TOKEN            | 30/30
-░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | Complete: ISSUE SECURITY TOKEN            | 0/30
-████████████████████████████████████████ | Submit  : CREATE CLAIM ISSUER IDENTITIES  | 5/5
-░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | Complete: CREATE CLAIM ISSUER IDENTITIES  | 0/5
-████████████████████████████████████████ | Submit  : ADD CLAIM ISSUERS               | 30/30
-░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | Complete: ADD CLAIM ISSUERS               | 0/30
-████████████████████████████████████████ | Submit  : MAKE CLAIMS                     | 30/30
-░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | Complete: MAKE CLAIMS                     | 0/30
-Total storage size delta: 2048KB
-Total number of failures: 0
-Transactions processed:
-	Block Number: 17		Processed: 97	Time (ms): 5987
-	Block Number: 18		Processed: 127	Time (ms): 6004
-	Block Number: 19		Processed: 32	Time (ms): 5997
-	Block Number: 20		Processed: 2	Time (ms): 6004
-	Block Number: 21		Processed: 32	Time (ms): 5995
-DONE
-Claims Batch Test
-Claim Batch Test Completed
-```
+$ npm run 0_create-_dentities
```

Comment:
```suggestion
$ npm run 0_create_identities
```

Reply:
This needs to be fixed on the `develop` branch first.